interpreter/llvm/src/tools/clang/lib/Serialization/ASTReader.cpp

```diff
@@ -2498,6 +2499,23 @@ ASTReader::ReadControlBlock(ModuleFile &F,
               (uint32_t)Record[Idx++]}}};
         auto ImportedFile = ReadPath(F, Record, Idx);
 
+        struct stat Buffer;
```

Comment:
When we discussed this last week, Richard was overall fine with the idea but told that we probably need to add hash comparison to check if ImportedFile hash and the relocated modulename matches or not, so that we don't load some random pcm which has the same name.